enhancements/security/security-profiles-operator.md

```diff
@@ -0,0 +1,649 @@
+---
+title: security-profiles-operator
+authors:
+  - "@jaormx"
+  - "@jhrozek"
+  - "@saschagrunert"
+reviewers:
+  - "@ashcrow"
+  - "@mrunalp"
+  - "@sttts"
+  - "@mrunalp"
+  - "@shawn-hurley"
+  - "@dmesser"
+approvers:
+  - TBD
```

Comment:
If this is still an active proposal, you'll need to update to the new template to bring in the required metadata. We need an approver assigned, and it needs to be linked to a Jira or Bugzilla ticket for tracking the implementation.

Reply:
Will do (probably on Monday).
Are there any requirements on the Jira ticket? Would https://issues.redhat.com/browse/CMP-1091 do? (This is our team's main epic tracking the work)

Reply:
Since we drop the "status" field here, the idea is to provide a reader with some place to track implementation progress, and an epic like CMP-1091 would do that just fine.

Reply:
I did the update locally, but can't push it at the moment - @JAORMX who opened the PR is on vacation and currently I don't have the push rights to his repo. I think he should be back next week, though.